fastly/acl_entry.go

```diff
@@ -24,22 +19,12 @@ type ACLEntry struct {
 	UpdatedAt *time.Time `mapstructure:"updated_at"`
 }
 
-// entriesByID is a sortable list of ACL entries.
-type entriesByID []*ACLEntry
-
-// Len implements the sortable interface.
-func (s entriesByID) Len() int {
-	return len(s)
-}
-
-// Swap implements the sortable interface.
-func (s entriesByID) Swap(i, j int) {
-	s[i], s[j] = s[j], s[i]
-}
+const aclEntriesPath = "/service/%s/acl/%s/entries"
 
-// Less implements the sortable interface.
-func (s entriesByID) Less(i, j int) bool {
-	return s[i].ID < s[j].ID
+// ACLEntriesPath is exposed primarily for use by the generic Paginator.
+// See ./paginator.go for details.
+func ACLEntriesPath(serviceID, aclID string) string {
+	return fmt.Sprintf(aclEntriesPath, serviceID, aclID)
 }
```

Comment:
Given that you have `GetACLEntries` now, do you need to export this?  (Ditto elsewhere in the PR.)  A related question: do you need to export `NewPaginator`?  It looks like all uses of it are in the same package.

Reply:
Inlined this now.